Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ghost:6.5.3-alpine
+
+# Add the Object Store storage adapter
+# We use the main branch of the repository
+ADD https://github.com/CodeForAfrica/ghost-object-store-storage-adapter.git#main content/adapters/storage/object-store
+
+# Install dependencies for the storage adapter
+WORKDIR /var/lib/ghost/content/adapters/storage/object-store
+RUN npm install --omit=dev
+
+# Return to the original working directory
+WORKDIR /var/lib/ghost
+
+# Start Ghost
```

Comment:
Finally, its the same CMD as the official image, we'd be safer relying on the default from the base image

Reply:
This is true. Fixed.